torch/_inductor/kernel/mm.py

```diff
@@ -133,14 +133,31 @@ def tuned_mm(mat1, mat2, *, layout=None):
             )
 
     if m * n != 0 and use_cutlass_template(layout):
-        cutlass_template = CUTLASSGemmTemplate([mat1, mat2], layout, alpha=1, beta=0)
+        cutlass_template = CUTLASSGemmTemplate(
+            [mat1, mat2], layout, alpha=1, beta=0, can_fuse_epilogue=False
+        )
+        # This will list only ops incapable of EVT fusion
         ops = cutlass_template.gen_ops()
         for op in ops:
             cutlass_template.maybe_append_choice(
                 choices,
                 op=op,
             )
-        log.debug("Added %d cutlass gemm configs.", len(ops))
+        cutlass_template_evt = CUTLASSGemmTemplate(
+            [mat1, mat2], layout, alpha=1, beta=0, can_fuse_epilogue=True
+        )
```

Comment:
This is based on the assumption that non-EVT supporting GEMM ops are still something we want to be able to support alongside EVT-supporting ones. In that case I have the option of either generating one CUTLASSGemmTemplate per op, or ( as it is now ) use one CUTLASSGemmTemplate for entire lists of ops ( to be chosen via autotuning ) but need to let the CUTLASSGemmTemplate instance know if it's ops support EVT fusion or not. Depending on that flag they may then allow node fusion or not, even if they don't know which specific op they represent.

Reply:
I see. I think, I understand the EVT / non-EVT distinction logic, just curious if it can be hidden within the `CUTLASSGemmTemplate`.

Reply:
It can be easily wrapped into a utility function, so we don't need to duplicate this code over and over. I will add that as a static method to CutlassGemmTemplate due to the close association.